magicctapipe/scripts/lst1_magic/lst1_magic_event_coincidence.py

```diff
@@ -560,12 +624,17 @@ def event_coincidence(input_file_lst, input_dir_magic, output_dir, config):
     event_data = event_data.astype({"obs_id": int, "event_id": int})
 
     # Save the data in an output file
-    Path(output_dir).mkdir(exist_ok=True, parents=True)
+    if os.path.splitext(output_dir)[1] == ".h5":
```

Comment:
the change looks fine, but it does not seem to be connected with time coincidence